tests/Makefile

```diff
@@ -113,3 +115,31 @@ realclean: clean clean-output
 
 java: .PHONY
 	make -C java
+
+# Run test-suite with ASAN
+asan:
```

Comment:
will anyone ever use these options here? might be an improvement to leave them out, either way minor point.